app/static/src/app/components/WodinPanels.vue

```diff
@@ -110,6 +144,41 @@ export default defineComponent({
 
     $dark-grey: #777;
 
+    $widthToleranceLeft: calc(100% / 8);
+    $widthToleranceRight: calc(100% - calc(100% / 4));
+
+    .drag-overlay {
+        position: fixed;
+        background-color: rgba($color: #000000, $alpha: 0.25);
+        z-index: 9999;
+        width: 100vw;
+        max-width: 100%;
+        height: 100vh;
+        left: 0;
+        top: 0;
+        margin-top: 3.5rem;
+        cursor: col-resize;
+    }
```

Comment:
I think this is what I am seeing now when resizing; the grey overlay with dotted lines where snap happens - this is a surprise! It feels like left in debug, what is the idea here?

Reply:
haha this was intended as a potential solution to @EmmaLRussell 's comment about some sort of indication that something is happening when you drag from a collapsed state, emma did say a couple of suggestions, maybe i can try another one